app/models.py

```diff
@@ -567,7 +567,18 @@ def from_json(cls, data):
         # validate json with marshmallow
         fields = data.copy()
 
-        fields["created_by_id"] = fields.pop("created_by")
+        fields["created_by_id"] = fields.pop("created_by", None)
+        fields["organisation_id"] = fields.pop("organisation", None)
+        fields["go_live_user_id"] = fields.pop("go_live_user", None)
+        fields.pop("safelist", None)
+        fields.pop("permissions", None)
+        fields.pop("service_callback_api", None)
+        fields.pop("users", None)
+        fields.pop("annual_billing", None)
+        fields.pop("inbound_api", None)
+        fields.pop("letter_logo_filename", None)
+        fields.pop("letter_contact_block", None)
+        fields.pop("email_branding", None)
```

Comment:
Are we modifying current behavior by removing these properties from the service? Are these usually populated with the lazy loaders and not actual properties? (hence kinda making these useless on JSON translation)

Reply:
@Moro-Code confirmed that these are SQLAlchemy lazy loaders usually sitting in these properties, until properties are accessed.